mesonbuild/coredata.py

```diff
@@ -850,6 +850,9 @@ def read_cmd_line_file(build_dir, options):
         # literal_eval to get it into the list of strings.
         options.native_file = ast.literal_eval(properties.get('native_file', '[]'))
 
+def cmd_line_options_to_string(options):
+    return {k: v if isinstance(v, str) else str(v) for k, v in options.cmd_line_options.items()}
```

Comment:
str(str)` is fast pathed in the python anyway, I think its clearer and easier if we skip the ternary and `isinstance` and just  do:

`{k: str(v) for k, v in options...}`

Reply:
Good point, I made that change now.